experimental.md

```diff
@@ -0,0 +1,21 @@
+This document specifies extensions to the core ESTree AST types to support ES proposals that are at least [stage 0 or above](https://github.com/tc39/ecma262). This document is subject to change and should not be seen as stable.
```

Comment:
Can we have slightly stronger wording here? Something mentioning that "may change at anytime"?